add-on/panels/js/roomViews.jsx

```diff
@@ -401,9 +401,15 @@ loop.roomViews = (function(mozL10n) {
       }
 
       // Automatically start sharing a tab now we're ready to share.
+      if (this.state.roomState === ROOM_STATES.MEDIA_WAIT &&
+          nextState.roomState === ROOM_STATES.JOINING) {
+        this.props.dispatcher.dispatch(new sharedActions.StartBrowserShare());
```

Comment:
Can you also make sure this works in the rejoin case? I'm thinking when the rejoin button is hit.